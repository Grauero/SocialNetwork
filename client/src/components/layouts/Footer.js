import React from 'react';

const Footer = () => (
  <footer className="bg-light text-dark mt-5 p-4 text-center">
    Copyright &copy; {new Date().getFullYear()} Network
  </footer>
);

export default Footer;
