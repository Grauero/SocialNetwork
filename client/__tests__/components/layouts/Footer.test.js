import React from 'react';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';

import Footer from '../../../src/components/layouts/Footer';

const component = mount(<Footer />);

it('matches snapshot', () => {
  expect(toJSON(component)).toMatchSnapshot();
});
