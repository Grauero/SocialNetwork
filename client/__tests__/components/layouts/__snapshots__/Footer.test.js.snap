// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Footer>
  <footer
    className="bg-light text-dark mt-5 p-4 text-center"
  >
    Copyright Â© 
    2019
     Network
  </footer>
</Footer>
`;
